### PR TITLE
NP basics completed

### PR DESCRIPTION
**Pull request by party13**

### CODE REVIEW

File: Week_1/Python_basic_with_numpy/Python Basics With Numpy v3.ipynb [hunk 1]
```diff
@@ -409,19 +452,33 @@
     "    \"\"\"\n",
     "    \n",
     "    ### START CODE HERE ### (≈ 1 line of code)\n",
-    "    v = None\n",
+    "    v = image.reshape((image.shape[0]*image.shape[1], image.shape[2]))\n",
```
Comment (egoliuk) [inline Week_1/Python_basic_with_numpy/Python Basics With Numpy v3.ipynb]: В проверке ожидается матрица размером 18 на 1, у вас матрица размером 9 на 2
Суть в том, чтобы матрицу размером m на n на k превратить в матрицу размером m*n*k на 1

File: Week_1/Python_basic_with_numpy/Python Basics With Numpy v3.ipynb [hunk 2]
```diff
@@ -853,19 +974,25 @@
     "    \"\"\"\n",
     "    \n",
     "    ### START CODE HERE ### (≈ 1 line of code)\n",
-    "    loss = None\n",
+    "    loss = sum(abs(y-yhat))\n",
```
Comment (egoliuk) [inline Week_1/Python_basic_with_numpy/Python Basics With Numpy v3.ipynb:986]: Возможно тут лучше явно обращаться к функциям numpy
loss =  np.sum(np.abs(y-yhat))
потому что "Explicit is better than implicit" https://www.python.org/dev/peps/pep-0020/#id3 

хотя и так тоже будет работать 
https://stackoverflow.com/questions/20953127/why-built-in-functions-like-abs-works-on-numpy-array

File: Week_1/Python_basic_with_numpy/Python Basics With Numpy v3.ipynb (hunk 1 above)
Comment (party13) [reply Week_1/Python_basic_with_numpy/Python Basics With Numpy v3.ipynb]: ты прав! исправил эту формулу

**Comment by vasylcf** [comment]
Все ок, ответы полностью совпадают с ожидаемыми по заданию.

**Review by vasylcf — APPROVED** [review]

**Review by egoliuk — APPROVED** [review]
